SGoopas/Assets/Scripts/Game/Enemies/EnemySpawner.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemySpawner : MonoBehaviour {
+    Vector3[] points;
+    public GameObject enemyPrefab;
+    private GameObject enemy;
```

Comment:
FloatingEnemy is stored for obvious reasons. the enemy game object is stored for the null check. Idk if components work in the same way for null checks, and I'd like to avoid having to do enemyScript.gameObject if they don't, but i'll test it out